src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -134,89 +80,89 @@
  * <h3>Usage examples</h3>
  * <h4>Inform the genotype assignment of NA12878 using the 1000G Euro panel</h4>
  * <pre>
- * java -jar GenomeAnalysisTK.jar \
- *   -T CalculateGenotypePosteriors \
- *   -R reference.fasta \
+ * ./gatk-launch \
+ *   CalculateGenotypePosteriors \
  *   -V NA12878.wgs.HC.vcf \
  *   -supporting 1000G_EUR.genotypes.combined.vcf \
- *   -o NA12878.wgs.HC.posteriors.vcf 
+ *   -O NA12878.wgs.HC.posteriors.vcf
  * </pre>
  *
  * <h4>Refine the genotypes of a large panel based on the discovered allele frequency</h4>
  * <pre>
- * java -jar GenomeAnalysisTK.jar \
- *   -T CalculateGenotypePosteriors \
- *   -R reference.fasta \
+ * ./gatk-launch \
+ *   CalculateGenotypePosteriors \
  *   -V input.vcf \
- *   -o output.withPosteriors.vcf
+ *   -O output.withPosteriors.vcf
  * </pre>
  *
  * <h4>Apply frequency and HWE-based priors to the genotypes of a family without including the family allele counts
  * in the allele frequency estimates the genotypes of a large panel based on the discovered allele frequency</h4>
  * <pre>
- * java -jar GenomeAnalysisTK.jar \
- *   -T CalculateGenotypePosteriors \
- *   -R reference.fasta \
+ * ./gatk-launch \
+ *   CalculateGenotypePosteriors \
  *   -V input.vcf \
- *   -o output.withPosteriors.vcf \
+ *   -O output.withPosteriors.vcf \
  *   --ignoreInputSamples
  * </pre>
  *
  * <h4>Calculate the posterior genotypes of a callset, and impose that a variant *not seen* in the external panel
  * is tantamount to being AC=0, AN=100 within that panel</h4>
  * <pre>
- * java -jar GenomeAnalysisTK.jar \
- *   -T CalculateGenotypePosteriors \
- *   -R reference.fasta \
+ * ./gatk-launch \
+ *   CalculateGenotypePosteriors \
  *   -supporting external.panel.vcf \
  *   -V input.vcf \
- *   -o output.withPosteriors.vcf \
+ *   -O output.withPosteriors.vcf \
  *   --numRefSamplesIfNoCall 100
  * </pre>
  *
  * <h4>Apply only family priors to a callset</h4>
  * <pre>
- * java -jar GenomeAnalysisTK.jar \
- *   -T CalculateGenotypePosteriors \
- *   -R reference.fasta \
+ * ./gatk-launch \
+ *   CalculateGenotypePosteriors \
  *   -V input.vcf \
  *   --skipPopulationPriors \
  *   -ped family.ped \
- *   -o output.withPosteriors.vcf
+ *   -O output.withPosteriors.vcf
  * </pre>
  *
  */
-@DocumentedGATKFeature( groupName = HelpConstants.DOCS_CAT_VARMANIP, extraDocs = {CommandLineGATK.class} )
-public class CalculateGenotypePosteriors extends RodWalker<Integer,Integer> {
+@CommandLineProgramProperties(
+        summary = "Given a VCF with genotype likelihoods from the HaplotypeCaller, UnifiedGenotyper, or another source which provides" +
+                  "unbiased genotype likelihoods, calculate the posterior genotype state and likelihood given allele frequency" +
+                  "information from both the samples themselves and input VCFs describing allele frequencies in related populations.",
```

Comment:
The input samples are not necessarily used, for instance if there aren't enough I believe.  Input VCFs are also optional.